packages/styled/test/prop-filtering.test.js

```diff
@@ -6,17 +6,16 @@ import styled from '@emotion/styled'
 
 test('composes shouldForwardProp on composed styled components', () => {
   const StyledDiv = styled('div', {
-    shouldForwardProp: prop => prop === 'forwardMe'
+    shouldForwardProp: prop => prop !== 'foo'
   })()
 
   const ComposedDiv = styled(StyledDiv, {
-    shouldForwardProp: () => true
+    shouldForwardProp: prop => prop !== 'bar'
```

Comment:
Why these changes? Was this test failing with the changes to styled here? If the original test here was failing, I think that's a problem.

Reply:
I don't think the previous version of this test was foo - it was supposed to test `shouldForwardProp` composition but `() => true` doesn't make any difference in regards to the final behavior, because this test would pass with it or without this `shouldForwardProp` as the composition uses `&&` so if we fold this `shouldForwardProp1(prop) && shouldForwardProp2(prop)` to `shouldForwardProp1(prop) && true` then all we really have is `shouldForwardProp1(prop)`